### DISCUSSION TITLE
Session invalidation when changing password

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#2028] [Questions & Answers]
Hi everybody, I've received this email:

<img width="1227" alt="Schermata 2022-02-25 alle 10 46 02" src="https://user-images.githubusercontent.com/909743/155702372-0c521177-22ae-49d5-bd19-1a8645bbd885.png">

do you think it makes sense? should it be considered a bug?

**Comment by elidrissidev** [comment]
In my opinion, old sessions should be invalidated after a password is changed. I think this may be possible without even re-authenticating the customer?

**Comment by addison74** [comment]
I think I read about such an issue not so long ago. may have even been in a PR. I personally consider that when someone changes the password the session must end and a new login would be required. I did not analyze in detail but this is the first idea that comes to me at the moment for security purposes.
